src/com/ms/silverking/cloud/dht/client/impl/OpSender.java

```diff
@@ -234,8 +235,8 @@ private void createMessages(AsyncOperationImpl[] asyncOpImpls, int startIndex, i
         // FUTURE - consider sending sooner, in creation loop
         for (MessageGroup messageGroup : messageGroups) {
             if (debug) {
-                System.out.println("OpSender sending:");
-                System.out.println(messageGroup);
+                Log.fineAsync("OpSender sending:");
```

Comment:
debug is off by default. Make Log.warningAsyncf so that the debug flag has an effect